src/components/buttons/Send/InputOptions.tsx

```diff
@@ -44,10 +45,16 @@ const InputOptions = ({
                <button onClick={onScanIconClick}>
                   <ScanIcon className='size-8 text-gray-500' />
                </button>
-               <button onClick={onUserIconClick}>
+               <button
+                  onClick={disableEcashButtons ? undefined : onUserIconClick}
+                  className={disableEcashButtons ? 'opacity-50 cursor-not-allowed' : ''}
+               >
                   <UserIcon className='w-6 h-6 text-gray-500' />
                </button>
-               <button onClick={onGiftIconClick}>
+               <button
+                  onClick={disableEcashButtons ? undefined : onGiftIconClick}
+                  className={disableEcashButtons ? 'opacity-50 cursor-not-allowed' : ''}
+               >
                   <GiftIcon className='w-6 h-6 text-gray-500' />
```

Comment:
can't you buy a gift with lightning?

Reply:
Yea if you are in mintless mode you can use your lightning wallet to buy a gift and send it as ecash, but that's still considered "sending ecash". Sending lightning means you are paying an external lighting invoice or lightning address, so you will never be sending a gift with that